x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/comparison_side/comparison_side.stories.tsx

```diff
@@ -0,0 +1,315 @@
+    diff_outcome: ThreeWayDiffOutcome.CustomizedValueCanUpdate,
+    merge_outcome: ThreeWayMergeOutcome.Merged,
+    has_update: true,
+    conflict: ThreeWayDiffConflict.SOLVABLE,
```

Comment:
With this one I wanted to check / showcase a real-life situation. A few changes in a setup guide - a few words here, a sentence there... What do you think about have 1 or 2 longer mocks to check a more real-life case?

Reply:
I agree it's beneficial during development phase in storybook to verify the component works with real multiline data. Though preparing and maintaining such data is tricky since spotting differences manually takes time. Let's use simplified abstract text later for testing and leave current text here unchanged.